static/js/actions/email.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+export const actionCreatorGenerator = (type: string) => (
+  (args: any) => args === undefined ? { type: type } : { type: type, payload: args }
+);
```

Comment:
yeah, i think it will definitely be useful elsewhere. there should also be a clean-enough way to extend it for actions that require specific number/types of arguments. if you want to move this (and the tests) into a util file as part of the follow-up issue, that's cool w/ me